modules/deployment/src/main/java/org/eclipse/xpanse/modules/deployment/deployers/terraform/TerraformResult.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+package org.eclipse.xpanse.modules.deployment.deployers.terraform;
+
+import java.util.Map;
+import lombok.Builder;
+import lombok.Data;
+import org.eclipse.xpanse.modules.models.service.common.enums.Csp;
+
+/**
+ * Data model for the Terraform command execution results.
+ */
+@Data
+@Builder
+public class TerraformResult {
```

Comment:
the webhook request body must be the same TerraformResult object from the generated classes.